cristin-import/src/test/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumerTest.java

```diff
@@ -956,7 +957,7 @@ void shouldPersistChannelRegistryExceptionReportWhenNsdCodeIsNotInChannelRegistr
         var cristinObject = CristinDataGenerator.randomObject(category.getValue());
         cristinObject.setSecondaryCategory(category);
         cristinObject.getBookOrReportMetadata().getCristinPublisher().setPublisherName(randomString());
-        cristinObject.getBookOrReportMetadata().getBookSeries().setNsdCode(randomInteger());
+        cristinObject.getBookOrReportMetadata().getBookSeries().setNsdCode(NOT_EXISTING_NSD_CODE);
```

Comment:
Fixing flaky test, randomInteger() has returned integer which was equal to present nsd code in channel register csv.